### ISSUE TITLE
Activate GitHub discussions and transfer questions

### ISSUE THREAD

**Issue by trallnag** [gruntwork-io/terragrunt#1501]
GitHub has been supporting a discussions tab for some time now (basically issues but in a forum like layout) and now it is also possible to transfer existing issues to discussions. I think this could be beneficial for this repository, like half of the open issues are questions.

https://docs.github.com/en/discussions/quickstart
https://github.com/github/feedback/discussions/2952

![Discussions-nextjs-with-categories](https://user-images.githubusercontent.com/24834206/105250967-f33e0d80-5b7a-11eb-8f55-83792f5b9f10.png)

**Comment by brikis98** [comment]
Perhaps a dumb question, but what is the advantage of using GitHub Discussions over Issues? The UI looks awfully similar... Is it that it's called "Discussions," and therefore more inviting to questions? Or that it supports categories?

**Comment by trallnag** [comment]
Yeah it's really the same just with a different layout. Just thinking that it might be a good way to deal with question "issues". It might also help with reducing similar questions because you don't really close discussions and so on.

Some projects tend to outright not answer questions and refer to mailing lists or so, projects like Terragrunt seem to keep questions open? 

Just wanted to give a heads up on this feature because it has been a bit under the radar. Though it is under active development as well (feature to move existing issues was re-added just a few days ago)

**Comment by brikis98** [comment]
Good point! The number of open issues does keep growing, in part because many are questions/discussions. That makes issues harder to navigate, so perhaps GitHub Discussions would help here. Thanks for pointing it out; I'll raise this with the team!
